docs/mine/lotus/split-markets-miners.md

```diff
@@ -68,51 +104,66 @@ export LOTUS_BACKUP_BASE_PATH=/tmp
 lotus-miner run
 ```
 
-## Splitting the `lotus-miner` monolith
-
-### 1. Creating a backup
+#### 1. Create a backup
 
 ```shell
 export LOTUS_BACKUP_BASE_PATH=/tmp
 lotus-miner backup /tmp/backup.cbor
 ```
```

Comment:
Shouldn't this be part of the prerequisites?

Reply:
I don't think the word prerequisites really makes sense, as backing up the miner is an integral part of the process. I'm just going to remove the prerequisites heading